client/src/components/clan/ForumEntry.jsx

```diff
@@ -9,24 +11,57 @@ import { Switch, Route } from 'react-router-dom';
 class ForumEntry extends React.Component {
```

Comment:
Both have a one to many relationship. One clan has many forums. Forums have many posts.